installer_scripts/install-yb-voyager

```diff
@@ -108,19 +126,23 @@ centos_main() {
 	check_binutils_version
 	output "Installing RPM dependencies."
 	$YUM which wget git gcc make 1>&2
-	$YUM perl perl-DBI perl-App-cpanminus 1>&2
-	$YUM mysql-devel 1>&2
 	$YUM https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm 1>&2 || true
 	$YUM postgresql14 1>&2
-	$YUM perl-ExtUtils-MakeMaker 1>&2
 
 	install_golang linux
-	centos_install_oracle_instant_clients
-	update_yb_voyager_bashrc
-	install_ora2pg
+	if [ $ONLY_PG == "false" ] ; then
+		$YUM perl perl-DBI perl-App-cpanminus 1>&2
+		$YUM mysql-devel 1>&2
+		$YUM perl-ExtUtils-MakeMaker 1>&2
+		centos_install_oracle_instant_clients
+		update_yb_voyager_bashrc
+		install_ora2pg
+	fi
 	install_yb_voyager
 	create_yb_session_vars_file
-	create_base_ora2pg_conf_file
+	if [ $ONLY_PG == "false" ] ; then
+		create_base_ora2pg_conf_file
+	fi
```

Comment:
Merge this if block into the earlier if block by moving line 144 after line 139.

Reply:
Actually I did this only earlier. But the `/etc/yb-voyager` is created in the `create_yb_session_vars_file`. I have now added a` mkdir -p /etc/yb-voyager` statement in `create_base_ora2pg_conf_file` too. Since in case of macos we only run `create_yb_session_vars_file`, I can't remove the `mkdir -p /etc/yb-voyager` from that function.